runtime/src/curated_grandpa.rs

```diff
@@ -21,13 +21,14 @@ use codec::Decode;
 use sr_primitives::traits::{As, Hash as HashT, BlakeTwo256, Zero};
 use rstd::prelude::*;
 
-
 pub trait Trait: grandpa::Trait {}
 
 decl_storage! {
 	trait Store for Module<T: Trait> as CuratedGrandpa {
 		/// How often to shuffle the GRANDPA sets.
-		pub ShufflePeriod get(shuffle_period) build(|_| T::BlockNumber::sa(1024u64)): T::BlockNumber;
+		///
+		/// 0 means never.
+		pub ShufflePeriod get(shuffle_period) config(shuffle_period): T::BlockNumber;
```

Comment:
Can't it just be `config()`?

Reply:
I wasn't sure if that would `snake_case` it correctly so better safe than sorry :P.